docs/sources/review/lint-prose/index.md

```diff
@@ -59,3 +59,45 @@ The task title contains a verb and an object. For example: "Create a dashboard".
 
 <!-- vale Grafana.Quotes = YES -->
 ```
+
+## Use Vale in VS Code
+
+You can use Vale to lint your current document in VS Code.
+
+1. Clone the [Technical Documentation](https://github.com/grafana/technical-documentation) repository.
+
+   ```bash
+   git clone git@github.com:grafana/technical-documentation.git
+   ```
+
+1. Download [Vale](https://github.com/errata-ai/vale/releases).
+1. Extract the downloaded zip file to `$HOME/vale`.
```

Comment:
I might be an idiot, but I don't know how to do this. I've gone into my top-level directory (AFAIK) on my machine and displayed hidden files and I've tried doing a search using terminal for a vale file, but I don't know where this should be.

Reply:
Fixed this up by adding macOS specific instructions